test/acceptor_SUITE.erl

```diff
@@ -597,7 +597,7 @@ misc_connection_alarms(_) ->
 		Self ! {connection_alarm, {Ref, AlarmName, length(ActiveConns)}}
 	end,
 	Alarms0 = #{
-		test1 => Alarm1 = #{type => num_connections, treshold => 2, cooldown => 0, callback => AlarmCallback},
+		test1 => Alarm1 = #{type => num_connections, threshold => 2, cooldown => 0, callback => AlarmCallback},
```

Comment:
We should probably keep a test that uses treshold.

Reply:
We do, see the next line after this one.

Reply:
Ahah understood. In that case a comment before the next line should say that we are testing for backward compatibility but that it should be removed in 3.0.

Reply:
Done in last commit.